src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('tbody');
+const rows = document.getElementsByTagName('tr');
+
+const appendRowButton = document.querySelector('.append-row');
+const deleteRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const deleteColumnButton = document.querySelector('.remove-column');
+
+const growthTable = () => {
+  appendRowButton.addEventListener('click', () => {
+    const lastRow = rows[rows.length - 1];
+    const cloneRows = lastRow.cloneNode(true);
+
+    table.appendChild(cloneRows);
+
+    if (rows.length >= 10) {
+      appendRowButton.setAttribute('disabled', '');
+    }
+
+    if (rows.length > 2) {
+      deleteRowButton.removeAttribute('disabled');
+    }
+  });
+
+  deleteRowButton.addEventListener('click', () => {
+    table.removeChild(rows[rows.length - 1]);
+
+    if (rows.length === 2) {
```

Comment:
The condition `rows.length === 2` assumes there are always at least two rows. If the table can have fewer than two rows, this logic might not work as expected.